Dockerfile.zkvm-demo

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:16.04 as build
+
+MAINTAINER Tom Llewellyn-smith <tom@stellar.org>
+
+RUN apt-get update && apt-get install -qy curl \
+        build-essential \
+        libpq-dev \
+        libsqlite3-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV PATH "/root/.cargo/bin:$PATH"
+
+# accept defaults
+RUN curl https://sh.rustup.rs -sSf > rustup-init.sh && \
+    bash rustup-init.sh -y && \
+    rustup install nightly-2019-08-19 && \
```

Comment:
can we read from `./demo/rust-toolchain` to install an appropriate nightly version?

Reply:
This is now complete.